src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/api/model/IkkeAktuellRequest.kt

```diff
@@ -14,13 +15,25 @@ data class IkkeAktuellRequest(
     val begrunnelse: String?,
     override val forslagId: UUID?,
 ) : EndringsforslagRequest {
-    private val kanBliIkkeAktuell = listOf(DeltakerStatus.Type.VENTER_PA_OPPSTART, DeltakerStatus.Type.IKKE_AKTUELL)
+    private val kanBliIkkeAktuell = listOf(
+        DeltakerStatus.Type.VENTER_PA_OPPSTART,
+        DeltakerStatus.Type.DELTAR,
+        DeltakerStatus.Type.IKKE_AKTUELL,
+    )
 
     override fun valider(deltaker: Deltaker) {
         validerAarsaksBeskrivelse(aarsak.beskrivelse)
         require(deltaker.status.type in kanBliIkkeAktuell) {
             "Kan ikke sette deltaker med status ${deltaker.status.type} til ikke aktuell"
         }
+        if (deltaker.status.type == DeltakerStatus.Type.DELTAR) {
+            require(statusForMindreEnn15DagerSiden(deltaker)) {
```

Comment:
Hva er grunnen til at det er bedre å bruke status i steden for `deltaker.startdato` her? (Jeg tror kanskje det er jeg som kanskje har skrevet dette opprinnelig så kanskje jeg burde svare på det... 😅)

Et problem som jeg kan se for meg med dette er vel om en deltaker f.eks blir forlenget etter å ha fått `HAR_SLUTTET`, da får de vel en ny `DELTAR` status, men det gir vel ikke mening å kunne sette denne deltakeren til `IKKE_AKTUELL` 🤔

Reply:
Nei og vi hadde et case på akkurat det du beskriver i prod forrige uke faktisk.. Begrunnelsen for å bruke statusdatoen i stedet for startdatoen er at hvis man setter startdatoen for langt tilbake i tid og så viser det seg at det var feil, så blir deltakelsen "låst" selv om endringen nettopp ble gjort. Hvis vi derimot ser på hvor lenge deltakeren har hatt statusen så vil man ikke få problemet med at man låser deltakeren for endring, og i tillegg kan man selv reaktivere deltakeren hvis man setter den til ikke aktuell ved et uhell. Lars og jeg diskuterte dette ifm det som skjedde i prod forrige uke, og landet på at det fortsatt er den løsningen med minst ulemper funksjonelt sett.